<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>595</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CamLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="YUVLabel" name="yuvLabel">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>5</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>916</x>
      <y>4</y>
      <width>192</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="clearMarks">
         <property name="text">
          <string>Clear Marks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="fpsLabel">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>484</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>324</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Visualization</string>
    </property>
    <widget class="QCheckBox" name="thresh_checkbox">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>20</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>threshold</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="blobs_checkbox">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>40</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>blobs</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>268</y>
      <width>109</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="camRadio">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Camera</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="arenaRadio">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Arena</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>776</x>
      <y>268</y>
      <width>161</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Bot Data</string>
    </property>
    <widget class="QLabel" name="botPosLabel">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>24</y>
       <width>129</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>x, y =</string>
     </property>
    </widget>
    <widget class="QLabel" name="botAngleLabel">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>44</y>
       <width>129</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Angle =</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>452</y>
      <width>191</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="arena_checkBox">
    <property name="geometry">
     <rect>
      <x>652</x>
      <y>408</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Arena Calibration</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>392</y>
      <width>177</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Mouse Coord</string>
    </property>
    <widget class="QLabel" name="mouseXLabel">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>24</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>x = </string>
     </property>
    </widget>
    <widget class="QLabel" name="mouseYLabel">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>48</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>y = </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>YUVLabel</class>
   <extends>QLabel</extends>
   <header location="global">yuvlabel.h</header>
  </customwidget>
  <customwidget>
   <class>CamLabel</class>
   <extends>QLabel</extends>
   <header location="global">camlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
